prime-router/settings/organizations.yml

```diff
@@ -190,13 +190,15 @@
       jurisdictionalFilter: [ "(%performerState.exists() and %performerState = 'IG') or (%patientState.exists() and %patientState = 'IG')" ]
       qualityFilter:
         - "(%performerState.exists() and %performerState = 'IG') or (%patientState.exists() and %patientState = 'IG')"
+      processingModeFilter:
+            - "(%performerState.exists() and %performerState = 'IG') or (%patientState.exists() and %patientState = 'IG')"
```

Comment:
Good callout, I am sorry I wasn't clearer. I will make sure to specify next time. Asking Arnej what org we will be using, but since the changes are going to master, I will revert this.

Reply:
For the connectathon, we can make new sender/receivers under the development ORG.